BridgeAppSDK/SBAPermissionsManager.m

```diff
@@ -237,18 +238,29 @@ - (NSString *)permissionDescriptionForType:(SBAPermissionsType)type {
         case SBAPermissionsTypeLocalNotifications:
             return NSLocalizedStringWithDefaultValue(@"SBA_REMINDER_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Allowing notifications enables the app to show you reminders for activity sessions.", @"");
         case SBAPermissionsTypeLocation:
-            return NSLocalizedStringWithDefaultValue(@"SBA_LOCATION_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Using your GPS enables the app to accurately determine distances travelled. Your actual location will never be shared.", @"");
+            return [self bundlePermissionDescriptionForKey: @"NSLocationWhenInUseUsageDescription" defaultValue:@"Using your GPS enables the app to accurately determine distances travelled. Your actual location will never be shared."];
         case SBAPermissionsTypeCoremotion:
-            return NSLocalizedStringWithDefaultValue(@"SBA_COREMOTION_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Using the motion co-processor allows the app to determine your activity, helping the study better understand how activity level may influence disease.", @"");
+            return [self bundlePermissionDescriptionForKey: @"NSMotionUsageDescription" defaultValue:@"Using the motion co-processor allows the app to determine your activity, helping the study better understand how activity level may influence disease."];
         case SBAPermissionsTypeMicrophone:
-            return NSLocalizedStringWithDefaultValue(@"SBA_MICROPHONE_PERMISSIONS_DESCRIPTION", nil, SBABundle(), @"Access to microphone is required for your Voice Recording Activity.", @"");
-        case SBAPermissionsTypeCamera:
+            return [self bundlePermissionDescriptionForKey: @"NSMicrophoneUsageDescription" defaultValue:@"Access to microphone is required for your Voice Recording Activity."];
         case SBAPermissionsTypePhotoLibrary:
+            return [self bundlePermissionDescriptionForKey: @"NSPhotoLibraryUsageDescription" defaultValue:@"Photo Library"];
+        case SBAPermissionsTypeCamera:
```

Comment:
SBAPermissionsTypePhotoLibrary and  SBAPermissionsTypeCamera were previously undocumented in AppCore with drop-through to "not supported"

Since I found a required key for the photo library, I added that to the list. Left Camera alone b/c I'm not sure what the proper permissions are for it. As long as we are continuing to use the permissions manager in it's current (less than awesome) format, I figured I would add an assert to alert devs that this key needs to be added to the permissions. This issue didn't show up with my previous on-device testing, but not sure if that was b/c I hadn't upgraded my phone from iOS 9 and was relying upon the simulator to catch iOS 10 issues.